DataFormats/GEMRecHit/src/classes_def.xml

```diff
@@ -36,6 +36,15 @@
   <class name="edm::Wrapper<edm::RangeMap<CSCDetId,edm::OwnVector<GEMCSCSegment,edm::ClonePolicy<GEMCSCSegment> >,edm::ClonePolicy<GEMCSCSegment> > >" splitLevel="0"/>
   <class name="GEMCSCSegmentRef" splitLevel="0"/>
 
+  <class name="GEMSegment" ClassVersion="-1">
+  <version ClassVersion="-1" checksum="1513163288"/>
```

Comment:
why "-1" ?
for sensible types we expect something with a class version starting from 2 (maybe even better from 3)